src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Random random = new Random();
+
+    public Ball getRandomBall() {
+        Ball newBall = new Ball(colorSupplier.getRandomColor(), random.nextInt(100));
+        return newBall;
```

Comment:
We should declare new variable if we plan to reuse it or when it's name adds clarity to complicated code.
When we have newBall = new Ball(...) it doesn't provide any benefits
```suggestion
        return new Ball(colorSupplier.getRandomColor(), random.nextInt(MAX_BALL_NUMBER));
```